### ISSUE TITLE
Flows: allow to select and use GRPC requests

### ISSUE THREAD

**Issue by TheSpy** [postmanlabs/postman-app-support#12207]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Flows do not allow adding GRPC requests, but only http requests.

### Describe the solution you'd like

I would like to be able to use GRPC requests within flows.

![image](https://github.com/postmanlabs/postman-app-support/assets/514031/da459504-2f76-4e95-8c79-3646db2df703)

**Comment by SimFG** [comment]
postflow is really attractive !!! hope it can support grpc requests

**Comment by arashrasoulzadeh** [comment]
#up

**Comment by behnambm** [comment]
This would be a really useful feature.

**Comment by yuyu-kong** [comment]
#up

**Comment by andrey-borue** [comment]
#up

**Comment by VelvetApparatus** [comment]
#up

**Comment by IvoLeal72** [comment]
#up

**Comment by rabbah** [comment]
noted!

**Comment by prempreetbrar** [comment]
#up

**Comment by michaeldowling** [comment]
Any updates on this feature?  Seems pretty ridiculous to have support for various kinds of requests like gRPC but yet only support HTTP in flows?

**Comment by rabbah** [comment]
agreed @michaeldowling... we will have more to say about this soon.

**Comment by krosnick** [comment]
Hi everyone, we hear you and we're working on adding support for gRPC in Flows. Will post here once it's shipped!

<img width="330" height="508" alt="Image" src="https://github.com/user-attachments/assets/3295445e-55c3-4998-b1a7-d255f3e9487a" />

**Comment by morphey83** [comment]
#up

**Comment by cyrielle-hada** [comment]
Hi everyone, gRPC requests are now supported in Flows. Currently this supports the unary method only. Give it a try and let us know if you run into any issues!

<img width="623" height="446" alt="Image" src="https://github.com/user-attachments/assets/749ca3c3-508b-402e-ba4e-44d27b74044d" />
